### PR TITLE
Add User agent prefix string

### PR DESCRIPTION
**Pull request by ad-astra-via**

### CODE REVIEW

**Review by omarmosid — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,6 +24,40 @@ Find out more about Managed Components [here](https://blog.cloudflare.com/zaraz-
...
+
+2.  To place an embed on a page using Cloudflare Zaraz, use a placeholder `twitter-post` HTML element with the following attributes:
+
+```html
+<twitter-post component="twitter" tweet-id="1754336034228171055"></twitter-post>
+```
+
+### Support
+
+This MC only supports the rendering an image, text, captions and post details. Posts that include an image gallery and/or videos will be rendered as a single image post.
```
Comment (simonabadoiu) [inline README.md]: typo here?

File: src/index.ts [hunk 2]
```diff
@@ -105,7 +104,9 @@ export default async function (manager: Manager) {
             headers: {
               accept:
                 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
-              'User-Agent': `${userAgent}`,
+              'User-Agent':
+                client?.userAgent ||
+                'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36',
```
Comment (simonabadoiu) [inline src/index.ts]: this default user agent string can live in a constant that will be used in both line 109 and 148

File: README.md (hunk 1 above)
Comment (ad-astra-via) [reply README.md]: Shoot actually this entire sentence is out of place.

File: src/index.ts (hunk 2 above)
Comment (ad-astra-via) [reply src/index.ts]: Indeed, thanks!

File: README.md [hunk 3]
```diff
@@ -24,6 +24,40 @@ Find out more about Managed Components [here](https://blog.cloudflare.com/zaraz-
 
 `txn_id` is the unique identifier of your Twitter Pixel. You can find it inside the Twitter snippet, as `twq('init','twitter_pixel_id');`. [Learn more](https://business.twitter.com/en/help/campaign-measurement-and-analytics/conversion-tracking-for-websites.html)
 
+## ⎙ Embeds
+
+### Twitter Post
+
+This Managed Components uses the [Embeds API](https://managedcomponents.dev/specs/embed-and-widgets/embeds) to render tweets on a web page, with its embed `twitter-post`. This Embed accepts the folowing parameters:
```
Comment (bjesus) [inline README.md]: the embed name is `post` I think, not `twitter-post`

Comment (bjesus) [reply README.md]: "Managed Components" should be singular

File: README.md [hunk 4]
```diff
@@ -24,6 +24,40 @@ Find out more about Managed Components [here](https://blog.cloudflare.com/zaraz-
...
+
+2.  To place an embed on a page using Cloudflare Zaraz, use a placeholder `twitter-post` HTML element with the following attributes:
+
+```html
+<twitter-post component="twitter" tweet-id="1754336034228171055"></twitter-post>
+```
+
+### Support
+
+This MC only supports the display of the tweet’s text, account and profile information, date and time, as well as likes, replies, and other engagement metrics.
```
Comment (bjesus) [inline README.md]: use "Managed Components" instead of MC

**Review by simonabadoiu — APPROVED** [review]

**Pull request merged**